cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/BuilderUtils.java

```diff
@@ -508,6 +516,34 @@ private static void assembleExecutable(BLangPackage bLangPackage, Path project)
                         copyFromJarToJar(importJar, uberJar);
                     }
                 }
+
+                // Iterate through .balo and copy the platform libs
+                String baloName = getFileName(bLangPackage.packageID.name.value);
```

Comment:
As I understood this only adds native libraries of the building module likewise we need to add libraries from imports and transitive dependencies as well.